quickcheck-dynamic/src/Test/QuickCheck/StateModel.hs

```diff
@@ -43,7 +43,7 @@ import Control.Monad
 import Control.Monad.Identity
 import Control.Monad.Reader
 import Control.Monad.State
-import Control.Monad.Writer (WriterT, runWriterT, tell)
+import Control.Monad.Writer (WriterT (..), runWriterT, tell)
```

Comment:
No longer needed?

Reply:
Thanks. I should not try to work on this while my mind is elsewhere...